Sources/Shout/SFTP.swift

```diff
@@ -65,6 +70,40 @@ public class SFTP {
         self.sftpSession = sftpSession
     }
 
+    /// Lists a directory content from the remote server
+    /// - Parameter remotePath: the path to the existing directory on the remote server to list
+    public func ls(remotePath: String) throws -> [String:LIBSSH2_SFTP_ATTRIBUTES]  {
```

Comment:
Would it be more user friendly to return `[String: FileTypes]`? Usually I think it's best if public funcs  avoid using libssh2 types directly and instead use nicer wrapper types

Reply:
Also since the other funcs in this class use more description names like `download` rather than `cp` or something like that, I'm inclined to think `func listFiles(in directory: String)` might be the more appropriate name